protected/humhub/config/web.php

```diff
@@ -15,23 +15,26 @@
     'components' => [
         'request' => [
             'class' => 'humhub\components\Request',
+            'csrfCookie' => ['httpOnly' => true, 'secure' => (isset(Yii::$app->request->isSecureConnection) ? false : true)],
         ],
         'response' => [
             'class' => 'humhub\components\Response',
         ],
         'user' => [
             'class' => 'humhub\modules\user\components\User',
             'identityClass' => 'humhub\modules\user\models\User',
+            'identityCookie' => ['name' => '_identity', 'httpOnly' => true, 'secure' => (isset(Yii::$app->request->isSecureConnection) ? false : true)],
             'enableAutoLogin' => true,
             'authTimeout' => 1400,
-            'loginUrl' => ['/user/auth/login']
+            'loginUrl' => ['/user/auth/login'],
         ],
         'errorHandler' => [
             'errorAction' => 'error/index',
         ],
         'session' => [
             'class' => 'humhub\modules\user\components\Session',
             'sessionTable' => 'user_http_session',
+            'cookieParams' => ['httpOnly' => true, 'secure' => (isset(Yii::$app->request->isSecureConnection) ? false : true)],
```

Comment:
instead `(isset(Yii::$app->request->isSecureConnection) ? false : true)` you can use `Yii::$app->request->isSecureConnection`